Makefile

```diff
@@ -149,9 +150,31 @@ destroy-and-recreate-database: purge-postgres setup
 devceleryworkers:
 	$(MAKE) -e DJANGO_SETTINGS_MODULE=contentcuration.dev_settings prodceleryworkers
 
-run-services:
+devrun-django:
+	python contentcuration/manage.py runserver --settings=contentcuration.dev_settings 0.0.0.0:8081
+
+devrun-server:
+	set -ex; \
+	function _on_interrupt() { $(DOCKER_COMPOSE) stop studio-nginx; }; \
+	trap _on_interrupt SIGINT SIGTERM SIGKILL ERR; \
+	$(DOCKER_COMPOSE) up -d studio-nginx; \
+	$(MAKE) -j 2 devrun-django devrun-webpack
```

Comment:
Maybe add an `echo` here to tell the actual URL to access on, to make sure people ignore the logging from the django runserver?